src/nwb2bids/_converters/_datalad_utils.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pathlib
+
+
+def _read_first_bytes(file_path: pathlib.Path, n: int = 6) -> bytes:
+    """
+    Read the first n (by default 6) bytes of a file; necessary for Windows 'symlinks'.
+    """
+    with file_path.open(mode="rb") as file_stream:
+        first_bytes = file_stream.read(n)
+        return first_bytes
+
+
+def _content_is_retrieved(file_path: pathlib.Path) -> bool:
+    """
+    Check if a file contents are fetched from the git-annex.
```

Comment:
you might like a clarification in the docstring and a test on behavior on broken symlinks